test/run_test.py

```diff
@@ -27,6 +26,23 @@
 executable = args.executable if args.executable else "/usr/bin/python2.7"
 os_release = args.release if args.release else "Fedora release 24 (Twenty Four)"
 
+#If environment path set, use rather that
+env_config_path = os.environ.get('RETRACE_SERVER_CONFIG_PATH')
+if env_config_path:
+    config_path = env_config_path
+else:
+    config_path = args.config_path if args.config_path else  "../src/config/retrace-server.conf"
+
+env_plugin_dir = os.environ.get('RETRACE_SERVER_PLUGIN_DIR')
+if env_plugin_dir:
+    plugins_dir = env_plugin_dir
+else:
+    plugins_dir = args.plugins_dir if args.plugins_dir else "../src/plugins/"
+
+#Add environment variables for plugins and configuration file
+os.environ["RETRACE_SERVER_PLUGIN_DIR"] = plugins_dir
+os.environ["RETRACE_SERVER_CONFIG_PATH"] = config_path
```

Comment:
Set these environment variables in the Makefile. If you do that, you don't need to introduce the argument variables.